### ISSUE TITLE
Linux download in releases

### ISSUE THREAD

**Issue by RustoMCSpit** [SpotlightKid/ykchorus#7]
There is only a windows download right now

**Comment by SpotlightKid** [comment]
There is a source distribution archive.

Providing binaries is what Linux distributions are for, in my opinion. The possible combinations of different architectures and libc and C++ standard library versions make it impractical to provide binaries, which work on all Linux distributions, and I don't have the resources to support that.

That said, here are binary packages for Arch Linux and derived distributions, built by our unofficial, third-party 'pro-audio' repo: https://arch.osamc.de/#x86_64/ykchorus

**Comment by RustoMCSpit** [comment]
> The possible combinations of different architectures
the vast majority use x86_64

**Comment by RustoMCSpit** [comment]
the description in the arch link is wrong Description:	A stereo dual-chorus audio effect LV2, LADSPA and VST2 plugin and JACK application

**Comment by SpotlightKid** [comment]
> the description in the arch link is wrong Description

Fixed in pkgrel 3.

**Comment by RustoMCSpit** [comment]
>  The possible combinations of different architectures 

the vast majority use x86_64

>  libc and C++ standard library versions

cant you just use the most recent? not following apologies.

i feel like your concern is not servicing 100 percent of Linux users when you could just settle and service 95 percent with ease, if im following

**Comment by SpotlightKid** [comment]
> the vast majority use x86_64

What about aarch64 (Raspberry Pi, Google Chrome, Zynthian, MOD Devices etc.)?

> cant you just use the most recent? 

No, because some distributions will have older lib/stdc++ libraries and a binary built on newer distro will not work there.

> not following apologies

I'm not sure I understand. Can you rephrase this?


The point is, even if I mange to provide binaries for 95% of Linux users, I'm pretty sure one of the remaining 5% will come and complain why there are no binaries for his/her system. And I just don't have the capacity to handle these issues across all my projects.

So my stance is: if you want Linux binaries, pressure your distribution (or "app store" or whatever) to provide them or build them yourself (which is not too hard, if you know the basics of Linux).

One reason for the existence of the Linux distribution ecosystem is to distribute the load of creating and providing binaries for all kinds of platforms. IMO distributions usually know best how to compile and package binaries the best way to integrate into their system and provide the best user experience (installers, signature checking, automated updates, etc.) (I admit that debian/Ubuntu sometimes mess around too much with upstream software and make things worse for developers and users).

I resist trends to create and mimic the same broken software installation landscape, which is prevalent on other OSes, where every software provider has their own way of installing their software and keep it up to date.

**Comment by RustoMCSpit** [comment]
> I'm not sure I understand. Can you rephrase this?

'i dont understand, sorry can you explain'

**Comment by SpotlightKid** [comment]
Oh, I think you meant to say:

"Apologies, I don't quite follow". Or, to put it simpler: "Sorry, I don't understand.".

:smiley:
